### ISSUE TITLE
Redirect this to whatwg/console?

### ISSUE THREAD

**Issue by sindresorhus** [DeveloperToolsWG/console-object#37]
This spec work looks more active: https://github.com/whatwg/console

**Comment by bkardell** [comment]
Yeah, I was going to work with @terinjokes on that -- it's been largely inactive too from what I can tell from https://github.com/whatwg/console/graphs/contributors.  But yeah, we need to merge these efforts, we've known that for a while - thanks for the reminder... Will try to get to that this weekend!
